internal/storage/bucket_handle.go

```diff
@@ -26,15 +26,15 @@ import (
 	"net/http"
 
 	"cloud.google.com/go/storage"
+	gcs2 "github.com/googlecloudplatform/gcsfuse/internal/storage/gcloud/gcs"
```

Comment:
we dont need gcloud/gcs under storage. just copy them directly into storage.
and for other packages remove gcs prefix eg: gcscaching -> caching

@sethiay - any other suggestions?

Reply:
No other suggestions for now.